spec/models/author_book_spec.rb

```diff
@@ -1,14 +1,17 @@
 require 'rails_helper'
 
 RSpec.describe AuthorBook do
-  # describe "validations" do
-  #   it { should validate_presence_of :book }
-  #   it { should validate_presence_of :author }
-  # end
+  #Not sure if validations are necessary
```

Comment:
Should we have this check for validation?

Reply:
Based on keeping the above code/relationship we should keep this test for coverage and verification